XamarinCommunityToolkit/Views/AvatarView/IColorTheme.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public interface IColorTheme
+	{
+		Color[] BackgroundColors { get; set; }
+
+		Color[] ForegroundColors { get; set; }
```

Comment:
I think these properties can be removed from the interface, right?

Reply:
Yeah they can go, the sealed class has them 😎